.env.example

```diff
@@ -1,8 +1,8 @@
 # Database - change to fit your database
 DB_HOST = 127.0.0.1 
 DB_USER = root
-DB_PASSWORD = db_password
-DB_NAME = db_name
+DB_PASSWORD = Hm4310913229
```

Comment:
Good thing you fixed this in a subsequent commit. Never publish passwords on GitHub :)